docs/ember/typing-your-dependencies.md

```diff
@@ -5,19 +5,34 @@ Here's an example that provides types for the `page-title` helper and the `Welco
 {% code title="types/global.d.ts" %}
```

Comment:
This section could use another reference to https://github.com/Gavant/glint-template-types.